readme.md

```diff
@@ -12,4 +12,6 @@ The build cannot be automated with GitHub Actions as it takes around 30 minutes
 
 It can be manually generated as follows:
 
-`C:\git\windows-docs-rs\crates\bindings>cargo doc --no-deps --target-dir ..\..\docs`
+```console
+C:\git\windows-docs-rs> cargo doc --no-deps -p windows -p bindings --target-dir docs
```

Comment:
Unfortunately `console` doesn't seem to parse and color cmd-like shells :/

Reply:
`pwsh` looks (only) a little bit better https://github.com/github/linguist/blob/32ec19c013a7f81ffaeead25e6e8f9668c7ed574/lib/linguist/languages.yml#L4359

Reply:
The main problem being, this is not exactly powershell either. The power in `console` is that it exists of three distinct parts: The prompt string (`user:path/ $`), the command highlighted as (ba)sh "script", and finally the output (flat color, but distinct from the prompt string and command):

```console
me:windows-docs-rs $ cd "crates/bindings" && cargo doc --no-deps -p windows -p bindings --target-dir docs
   Compiling proc-macro2 v1.0.26
   Compiling syn v1.0.71
   Compiling unicode-xid v0.2.2
   Compiling const-sha1 v0.2.0
    Checking quote v1.0.9
  Building [=============>             ] 12/23: syn, syn
```

That looks quite pretty doesn't it? :D